test/e2e/handler/nnce_conditions_test.go

```diff
@@ -128,23 +181,36 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 					Type:   shared.NodeNetworkConfigurationEnactmentConditionMatching,
 					Status: corev1.ConditionTrue,
 				},
-			}
-			for _, node := range nodes {
-				By(fmt.Sprintf("Check %s failing state is reached", node))
-				enactmentConditionsStatusEventually(node).Should(ConsistOf(failingEnactmentConditions...), "should eventually reach failing conditions at enactments")
+				shared.Condition{
+					Type:   shared.NodeNetworkConfigurationEnactmentConditionAborted,
+					Status: corev1.ConditionTrue,
+				},
 			}
 			By("Check policy is at degraded state")
 			waitForDegradedTestPolicy()
 
-			By("Check that the enactment stays in failing state")
+			for _, node := range nodes {
+				By(fmt.Sprintf("Check %s failing or aborted state is reached", node))
+				enactmentConditionsStatusEventually(node).Should(
+					SatisfyAny(
+						ConsistOf(failingEnactmentConditions...),
+						ConsistOf(abortedEnactmentConditions...),
+					), "should eventually reach failing of aborted conditions at enactments")
+			}
+
+			By("Check that the enactment stays in failing or aborted state")
```

Comment:
This will pass is something is broken and we have two failures and one aborted at sequencial rollout we will have just one failed enactment.

Reply:
Addressed by adding a function that counts failing and aborted enactments and expects 1 failing and len(nodes)-1 aborted enactments